lib/fastlane/plugin/browserstack/helper/browserstack_helper.rb

```diff
@@ -0,0 +1,38 @@
+require 'fastlane_core/ui/ui'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+
+  module Helper
+    class BrowserstackHelper
+      # class methods that you define here become available in your action
+      # as `Helper::BrowserstackHelper.your_method`
+      #
+      def self.show_message
+        UI.message("Hello from the browserstack plugin helper!")
+      end
+
+      # Uploads file to BrowserStack
+      # Params :
+      # +browserstack_username+:: BrowserStack's username.
+      # +browserstack_access_key+:: BrowserStack's access key.
+      # +file_path+:: Path to the file to be uploaded.
+      # +api_endpoint+:: BrowserStack's api endpoint, either AppAutomate or AppLive url.
+      def self.upload_file(browserstack_username, browserstack_access_key, file_path, api_endpoint)
+        url = "https://" + browserstack_username + ":" + browserstack_access_key + "@" + api_endpoint
+
+        begin
+          response = RestClient.post(url, file: File.new(file_path, 'rb'))
```

Comment:
Use something like [this](https://www.krautcomputing.com/blog/2015/06/21/how-to-use-basic-authentication-with-the-ruby-rest-client-gem/) for auth